### PR TITLE
Fix get_samples function 

### PR DESCRIPTION
**Pull request by tango4j**
# What does this PR do ?

Update and remove unnecessary parts in `get_samples` function. "int" type setting is no longer needed. 

Removed `get_samples` function from [decoder_timestamps_utils.py](https://github.com/NVIDIA/NeMo/pull/5141/files#diff-6e1688943c976f39bdee5fed0c8948f57c88411cfb69548b7fece4b9f7720912) since it is almost the same as get_samples function in [streaming_utils.py](https://github.com/NVIDIA/NeMo/blob/a17a69dea4f8768c0c444573c5a5c499c1e6e2ef/nemo/collections/asr/parts/utils/streaming_utils.py)

**Collection**: 
ASR

# Changelog 

- Remove unnecessary `int16` dtype and float conversion lines
- Keep the output data format as float
- Update librosa resample argument requirements

# Usage

Check out streaming_asr tutorial notebook.

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed.

### CODE REVIEW

File: nemo/collections/asr/parts/utils/decoder_timestamps_utils.py [hunk 1]
```diff
@@ -27,7 +25,7 @@
 from nemo.collections.asr.metrics.wer_bpe import WERBPE, CTCBPEDecoding, CTCBPEDecodingConfig
 from nemo.collections.asr.models import EncDecCTCModel, EncDecCTCModelBPE
 from nemo.collections.asr.parts.utils.speaker_utils import audio_rttm_map, get_uniqname_from_filepath
-from nemo.collections.asr.parts.utils.streaming_utils import AudioFeatureIterator, FrameBatchASR
+from nemo.collections.asr.parts.utils.streaming_utils import AudioFeatureIterator, FrameBatchASR, get_samples
```
Comment (VahidooX) [inline nemo/collections/asr/parts/utils/decoder_timestamps_utils.py:28]: How about move this method to a more general place? like audio_utils.py?

File: nemo/collections/asr/parts/utils/streaming_utils.py [hunk 2]
```diff
@@ -441,16 +442,28 @@ def __len__(self):
...
+    """
     with sf.SoundFile(audio_file, 'r') as f:
-        dtype = 'int16'
         sample_rate = f.samplerate
-        samples = f.read(dtype=dtype)
+        samples = f.read()
         if sample_rate != target_sr:
-            samples = librosa.core.resample(samples, sample_rate, target_sr)
-        samples = samples.astype('float32') / 32768
+            samples = librosa.core.resample(samples, orig_sr=sample_rate, target_sr=target_sr)
```
Comment (VahidooX) [inline nemo/collections/asr/parts/utils/streaming_utils.py:463]: So there is no need for reading in int type anymore?

File: tutorials/asr/Streaming_ASR.ipynb [hunk 3]
```diff
@@ -309,12 +309,10 @@
...
                 "def get_samples(audio_file, target_sr=16000):\n",
                 "    with sf.SoundFile(audio_file, 'r') as f:\n",
-                "        dtype = 'int16'\n",
                 "        sample_rate = f.samplerate\n",
-                "        samples = f.read(dtype=dtype)\n",
+                "        samples = f.read()\n",
                 "        if sample_rate != target_sr:\n",
-                "            samples = librosa.core.resample(samples, sample_rate, target_sr)\n",
-                "        samples=samples.astype('float32')/32768\n",
+                "            samples = librosa.core.resample(samples, orig_sr=sample_rate, target_sr=target_sr)\n",
```
Comment (VahidooX) [inline tutorials/asr/Streaming_ASR.ipynb:315]: We have one more of get_sample in this tutorials:
buffered_transducer_inference

**Review by VahidooX — CHANGES_REQUESTED** [review]

File: nemo/collections/asr/parts/utils/streaming_utils.py (hunk 2 above)
Comment (tango4j) [reply nemo/collections/asr/parts/utils/streaming_utils.py:463]: No we don't need to as far as I figured.

**Review by VahidooX — APPROVED** [review]
LGTM, thanks!

**Pull request merged**